llvm_ir_dataset_utils/compile_time_analysis_tools/datatset_download/create_tar.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -o errexit
```

Comment:
This looks like it just turns everything into a tar file?

We should be able to do this from within Python with the `tarfile` library.